diff --git a/node_modules/next/dist/next-server/server/api-utils.js b/node_modules/next/dist/next-server/server/api-utils.js
index ed6cdb1..fb56d81 100644
--- a/node_modules/next/dist/next-server/server/api-utils.js
+++ b/node_modules/next/dist/next-server/server/api-utils.js
@@ -15,6 +15,7 @@ async function apiResolver(req, res, params, resolverModule, apiContext, onError
     try {
         let config = {};
         let bodyParser = true;
+        let externalServer = false;
         if (!resolverModule) {
             res.statusCode = 404;
             res.end('Not Found');
@@ -25,6 +26,9 @@ async function apiResolver(req, res, params, resolverModule, apiContext, onError
             if (config.api && config.api.bodyParser === false) {
                 bodyParser = false;
             }
+            if (config.api && config.api.externalServer) {
+                externalServer = true;
+            }
         }
         // Parsing of cookies
         setLazyProp({ req: apiReq }, 'cookies', getCookieParser(req));
@@ -49,7 +53,7 @@ async function apiResolver(req, res, params, resolverModule, apiContext, onError
         }
         // Call API route method
         await resolver(req, res);
-        if (process.env.NODE_ENV !== 'production' && !utils_1.isResSent(res) && !wasPiped) {
+        if (process.env.NODE_ENV !== 'production' && !externalServer && !utils_1.isResSent(res) && !wasPiped) {
             console.warn(`API resolved without sending a response for ${req.url}, this may result in stalled requests.`);
         }
     }
